backendCore/dl/dl_model.py

```diff
@@ -0,0 +1,28 @@
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+
+
+class DLModel(nn.Module):
```

Comment:
Should we also update this [unit test](https://github.com/DSGT-DLP/Deep-Learning-Playground/blob/nextjs/tests/test_model.py) since this `backendCore/dl/dl_model.py` seems to be a duplicate version of `backend/dl/dl_model.py`?